src/screens/Asl/components/AslTable/index.js

```diff
@@ -56,6 +57,83 @@ const AslTable = props => {
     const [showVideo, setShowVideo] = useState(false);
     const [videoUrl, setVideoUrl] = useState('');
 
+    const wrapStyle = {
+      left: 100,
+      top: 100,
+      width: 500,
+      height: 500
+    }
+    const [style, setStyle] = useState({
+        left: 100,
+        top: 100,
+        width: 400,
+        height: 200
+    })
+    // 初始数据， 因为不需要重新render 所以用 useRef
+    const oriPos = useRef({
+        top: 0, // 元素的坐标
```

Comment:
English is the common language for coding; It's okay to have other languages but there must be an  English version.
// 初始数据， 因为不需要重新render 所以用 useRef
// The initial data, because it does not need to be re-rendered, use useRef
// 初始数据， 因为不需要重新render 所以用 useRef
const oriPos = useRef({
top: 0, // The coordinates of the element  (元素的坐标)
left: 0, 
cX: 0, // coordinates of the mouse  (鼠标的坐标)
cY: 0

Reply:
I think I removed Chinese comment for the latest version. Is that latest version?